blog/forms.py

```diff
@@ -0,0 +1,12 @@
+"""This module keeps the collection of form declarations representing HTML forms."""
+
+from django import forms
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+    """Form for Post model."""
```

Comment:
No. It's a form for Post entity. Don't write obvious things I can see two lines below.